packages/cli/src/options/beaconNodeOptions/execution.ts

```diff
@@ -1,15 +1,27 @@
+import fs from "node:fs";
 import {defaultOptions, IBeaconNodeOptions} from "@chainsafe/lodestar";
 import {ICliCommandOptions} from "../../util";
 
 export type ExecutionEngineArgs = {
   "execution.urls": string[];
   "execution.timeout": number;
+  "jwt-secret"?: string;
 };
 
 export function parseArgs(args: ExecutionEngineArgs): IBeaconNodeOptions["executionEngine"] {
+  let jwtSecretHex;
+  if (args["jwt-secret"]) {
```

Comment:
maybe extract this to a separate util so that we can have unit test for it

Reply:
added here: https://github.com/ChainSafe/lodestar/pull/3777/commits/53a3f16eb905ded7b98addf6004974a52934a579 :+1: